index.js

```diff
@@ -285,8 +326,10 @@ PersonaClient.prototype.validateHTTPBearerToken = function (request, response, n
     if (arguments.length > 3) {
         throw "Usage: validateHTTPBearerToken(request, response, next)";
     }
-    var token = this.getToken(request);
-    this.validateToken(token, request.param("scope"), function (error, validationResult) {
+    var token = this.getToken(request),
+        xRequestId = this.getXRequestId(request);
```

Comment:
Our agreed linting rules of choice (AirBnB) require individual `var` declarations (not comma-separated).